src/components/ProfileDevIcons/index.tsx

```diff
@@ -27,7 +30,7 @@ const ProfileDevIcons = () => {
           — <span style={{ whiteSpace: 'nowrap' }}>is my jam.</span>
           <br />
           Here are some of the technologies I've enjoyed hacking on lately:
-        </p>
+        </Text>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 10 spaces but found 8. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)